deploy/stacks/pipeline.py

```diff
@@ -1013,42 +975,3 @@ def user_guide_pre_build_alb(self, repository_name):
             role=self.expanded_codebuild_role.without_policy_updates(),
             vpc=self.vpc,
         )
-
-    def set_release_stage(
-        self,
-    ):
-        self.pipeline.add_wave(
-            f'{self.resource_prefix}-{self.git_branch}-release-stage'
-        ).add_post(
-            pipelines.CodeBuildStep(
-                id='GitRelease',
-                build_environment=codebuild.BuildEnvironment(
-                    build_image=codebuild.LinuxBuildImage.AMAZON_LINUX_2_5,
-                ),
-                partial_build_spec=codebuild.BuildSpec.from_object(
-                    dict(
-                        version='0.2',
-                        phases={
-                            'build': {
-                                'commands': [
-                                    'set -eu',
-                                    f'aws codeartifact login --tool pip --repository {self.codeartifact.codeartifact_pip_repo_name} --domain {self.codeartifact.codeartifact_domain_name} --domain-owner {self.codeartifact.domain.attr_owner}',
-                                    'python -m venv env',
-                                    '. env/bin/activate',
-                                    'pip install git-remote-codecommit',
-                                    'mkdir release && cd release',
-                                    f'git clone codecommit::{self.region}://dataall',
-                                    'cd dataall',
-                                    f'git checkout {self.git_branch}',
-                                    f'make version-minor branch={self.git_branch}',
```

Comment:
should you also remove the `Makefile` target?

Reply:
Good idea. Done!